manifests/t/Toit/Jaguar/1.0.5/Toit.Jaguar.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Toit.Jaguar
+PackageVersion: 1.0.5
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
Is this an x64 application? I see this in the logs, and it seems to install fine on an x64 sandbox:
```
Exception trying to launch executable C:\Users\Validator\AppData\Local\Programs\jaguar\jag.exe: System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)
```

---

Any update on this being x64?

---

> Any update on this being x64?

Haven't heard back from PR author; I'll check out the branch and see if a separate PR will pass without issue

---

Sorry for the confusion.
The x86 was inferred by wingetcreate. I have uploaded a new PR: https://github.com/microsoft/winget-pkgs/pull/46680